src/platform/auth/session/useSessionCookie.ts

```diff
@@ -15,7 +15,7 @@ export const useSessionCookie = () => {
     if (!isCloud) return
 
     const authStore = useFirebaseAuthStore()
-    const authHeader = await authStore.getAuthHeader()
+    const authHeader = await authStore.getAuthHeader(true)
```

Comment:
I don't think we always want to refresh. That's a lot of network traffic that we will send to firebase.

How long is the session? Maybe try keeping an interval (useInterval or useIntervalFn) thats a little shorter than the firebase expiration time?